core/templates/dev/head/pages/exploration-editor-page/services/exploration-property.service.spec.ts

```diff
@@ -61,7 +61,7 @@ describe('Exploration Property Service', function() {
   it('should create a new exploration properties object', function() {
     expect(function() {
       ExplorationPropertyService.init('initial value');
-    }).toThrow('Exploration property name cannot be null.');
+    }).toThrow(new Error('Exploration property name cannot be null.'));
```

Comment:
Here and elsewhere, I think you can use toThrowError?

Reply:
i fixed those file in my recent commit
thanks